File: maxon_epos_example/node/sample.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
# FileName:     sample
# CreatedDate:  2019-06-05 18:25:22
#

import rospy
from std_msgs.msg import Float32MultiArray


if __name__ == "__main__":
    rospy.init_node("sample", anonymous=True)
    pub = rospy.Publisher("/maxon_bringup/all_position", Float32MultiArray, queue_size=1)
    pub.publish(Float32MultiArray(data=[10000, -20000]))

File: maxon_epos_example/node/sample.cpp
/**
 * @file   sample
 * @brief  
 * @author arwtyxouymz
 * @date   2019-06-05 23:51:24
 */

#include <rclcpp/rclcpp.hpp>
#include "std_msgs/msg/Float32MultiArray.hpp"

int main(int argc, char** argv)
{
    rclcpp::init(argc, argv);
    rclcpp::Node::SharedPtr nh("sample");

    rclcpp::Publisher::SharedPtr pub = nh->create_publisher<std_msgs::msg::Float32MultiArray>("/maxon_bringup/all_position", 1000);
    std_msgs::msg::Float32MultiArray msg = std_msgs::msg::Float32MultiArray();
    msg.data.push_back(300000);
    msg.data.push_back(-200000);
    
    rclcpp::Rate sleep_rate(50);
    while (rclcpp::ok()) {
        pub.publish(msg);
        sleep_rate.sleep();
    }
    
    return 0;
}
